database/factories/FlightFactory.php

```diff
@@ -18,12 +18,15 @@ class FlightFactory extends Factory
      */
     public function definition(): array
     {
+        $airline = Airline::factory()->hasCities(2)->create();
```

Comment:
```suggestion
        $airline = AirlineFactory::new()->hasCities(2)->create();
```
Let's use the factory class directly instead of using it through a model's `factory` method. [Here's](https://light-it.slack.com/archives/C052JJZEPFH/p1684240120303009) the why. Make this change throughout the entire codebase 🙏🏻.